discopy/cat.py

```diff
@@ -767,6 +767,38 @@ def from_tree(cls, tree):
         return cls(terms=terms, dom=dom, cod=cod)
 
 
+class Frame(Box):
+    """ A box that has diagrams inside it. """
```

Comment:
Just make a dummy change CI should run again

```suggestion
    """ A box that has a list of diagrams inside it. """
```